modules/polygon/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@math.gl/polygon",
+  "description": "Polygon/polyline processing utilities",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "3.1.3",
```

Comment:
Master is now on 3.2. alpha, but I suspect you'll backport this to 3.1 branch?

Reply:
I don't need this on 3.1.